pkg/auth/util/provider_util_test.go

```diff
@@ -0,0 +1,46 @@
+package util_test
+
+import (
+	"github.com/rancher/rancher/pkg/auth/util"
+	"testing"
+)
+
+func TestEscapeUUID(t *testing.T) {
+	type args struct {
```

Comment:
You don't need this wrapper type, since you pass the string directory to the function under test.